src/components/Charts/ChartRt.tsx

```diff
@@ -0,0 +1,298 @@
+import React from 'react';
+import moment from 'moment';
+import { isUndefined } from 'lodash';
+import { min as d3min, max as d3max } from 'd3-array';
+import { Group } from '@vx/group';
+import { ParentSize } from '@vx/responsive';
+import { scaleLinear, scaleTime } from '@vx/scale';
+import { GridRows } from '@vx/grid';
+import { curveNatural } from '@vx/curve';
+import { AxisBottom, AxisLeft } from '@vx/axis';
+import { LinePath, Area } from '@vx/shape';
+import { useTooltip } from '@vx/tooltip';
+import { localPoint } from '@vx/event';
+import { LevelInfoMap, Level } from 'common/level';
+import { Column, RtRange, RT_TRUNCATION_DAYS } from 'common/models/Projection';
+import { CASE_GROWTH_RATE_LEVEL_INFO_MAP } from 'common/metrics/case_growth';
+import BoxedAnnotation from './BoxedAnnotation';
+import HoverOverlay from './HoverOverlay';
+import RectClipGroup from './RectClipGroup';
+import {
+  formatDecimal,
+  getChartRegions,
+  getTruncationDate,
+  getZoneByValue,
+  last,
+} from './utils';
+import * as Style from './Charts.style';
+
+type PointRt = Omit<Column, 'y'> & {
```

Comment:
hmm i'm not familiar with this syntax

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk - it allows omit one property from the type - in this case, `y` is `any` in `Column`, which is not compatible with `RtRange`